From 2ba3992e586692e283215de4b7f52bb3153c83a1 Mon Sep 17 00:00:00 2001
From: Peter Lohmer <peterlohmer01@gmail.com>
Date: Wed, 21 Nov 2018 12:27:33 +0100
Subject: [PATCH 5/7] wm3000u.cpp: bugfix in state machine when initializing
 and querying adjustment data version

Signed-off-by: Peter Lohmer <peterlohmer01@gmail.com>
---
 wm3000u.cpp |    1 +
 1 Datei geändert, 1 Zeile hinzugefügt(+)

diff --git a/wm3000u.cpp b/wm3000u.cpp
index 7ce3c27..ce1242a 100644
--- a/wm3000u.cpp
+++ b/wm3000u.cpp
@@ -333,6 +333,7 @@ void cWM3000U::ActionHandler(int entryAHS)
             AHS++;
             m_ActTimer->start(0,wm3000Continue); // event, damit statemachine weiterläuft
         }
+        break;
     }
 
     case InitializationOpenChannel0:
-- 
1.7.10.4

